R/event.R

```diff
@@ -28,7 +30,7 @@
 #'  
 #' @export
 dyEvent <- function(dygraph, 
-                    date,
+                    x,
```

Comment:
I that's a very good idea. Perhaps also print a deprecation warning if `!missing("date")` (e.g. "The date parameter is deprecated, please use the `x` parameter instead"). Then we can remove the date parameter entirely after a transition period.